<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculator</class>
 <widget class="QMainWindow" name="calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="firstNumberLne">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondNumberLne">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="firstNumberLbl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>First Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondNumberLbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Second Number:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sumBtn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Sum</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLbl">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="resultLcd">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
